integration/storagetest/logtests.go

```diff
@@ -461,8 +461,8 @@ func (*logTests) TestDequeueLeaves(ctx context.Context, t *testing.T, s storage.
 			if err != nil {
 				t.Fatalf("Failed to dequeue leaves (second time): %v", err)
 			}
-			if len(leaves) != 0 {
-				t.Fatalf("Dequeued %d leaves but expected to get %d", len(leaves), leavesToInsert)
+			if want := 0; len(leaves) != want {
+				t.Fatalf("Dequeued %d leaves but expected to get %d", len(leaves), want)
```

Comment:
Could you make this use the `got, want := len(leaves), 0; got != want` pattern? And use `got` in the error message?

Reply:
Yup, done.